<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Aleksandar" failures="2" tests="2" name="HLV App Tests" time="1.658" errors="0" timestamp="2021-09-15T16:08:02 CEST">
  <testcase classname="local.HLVTestApp.APITests.API_RegisterUserTests" name="API_RegisterUser" time="1.586">
    <failure type="java.lang.AssertionError" message="expected [201] but found [400]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [400]
at local.HLVTestApp.APITests.API_RegisterUserTests.API_RegisterUser(API_RegisterUserTests.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- API_RegisterUser -->
  <testcase classname="local.HLVTestApp.APITests.API_RegisterUserTests" name="API_RegisterUser" time="1.624">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at local.HLVTestApp.listeners.TestListener.onTestFailure(TestListener.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- API_RegisterUser -->
</testsuite> <!-- HLV App Tests -->
